cmd/clusterctl/client/init.go

```diff
@@ -108,6 +112,10 @@ func (c *clusterctlClient) Init(options InitOptions) ([]Components, error) {
 	// - There should be only one instance of the same provider.
 	// - All the providers must support the same API Version of Cluster API (contract)
 	if err := installer.Validate(); err != nil {
+		if options.IgnoreValidationErrors {
+			log.Error(err, "Ignoring validation errors")
+			return nil, nil
```

Comment:
What would be a use case for this feature?

Right now, it looks like if a user passes `--validate=false` it just print the validation errors along with `Ignoring valdiation errors` and does not conitnue with init anyway (same as before).

Reply:
I think you found a bug. The idea is probably to continue on l.121+

Reply:
If you really want to know, we have clusterctl embedded in VMware's Tanzu Kubernetes Grid, and I want the ability to stand up a bootstrap cluster with all our custom stuff, let a user do some additional fiddling, like say, put a custom clusterclass and associated bits, and then continue running. I'm doing this mostly in development to be fair.

We had another instance where some consumers are wrapping our CLI and then attempting to retry it in an idempotent fashion.

Anyway, i have fixed the above.

Reply:
Mostly wanted to know to figure out if the current behavior is a bug or expected :). But thx for the additional context

Reply:
It was a fair question! My answer was more meant in a "you really don't want to know" kind of way.